release.sh

```diff
@@ -16,10 +16,7 @@ build_dp_check() {
   file dp_check | grep 'statically linked'
   if [ $? != 0 ]; then
     echo "dp_check binary is not statically linked"
-  else
```

Comment:
nit: while we're here. Can we exit non-zero in this cast (if not statically linked) ?